test/integration/ads_integration_test.cc

```diff
@@ -1546,4 +1546,42 @@ TEST_P(AdsClusterV2Test, DEPRECATED_FEATURE_TEST(TypeUrlAnnotationRegression)) {
   test_server_->waitForCounterGe("cluster_manager.cds.update_rejected", 1);
 }
 
+class AdsSetNodeAlwaysIntegrationTest : public AdsIntegrationTest {
```

Comment:
This seems like a bit much boilerplate for not much gain, could we just write this test as regular test case within `AdsIntegrationTest` where we call `addConfigModifier` before `initialize`?

Reply:
Sounds good to me